modin/core/storage_formats/pandas/parsers.py

```diff
@@ -784,28 +784,77 @@ class ParquetFileToRead(NamedTuple):
 class PandasParquetParser(PandasParser):
     @staticmethod
     def _read_row_group_chunk(
-        f, row_group_start, row_group_end, columns, engine, to_pandas_kwargs
+        f, row_group_start, row_group_end, columns, filters, engine, to_pandas_kwargs
     ):  # noqa: GL08
         if engine == "pyarrow":
-            from pyarrow.parquet import ParquetFile
+            if filters is not None:
```

Comment:
As mentioned above, these do not always behave the same in Pandas (because they do not behave the same in the underlying libraries), so I would prefer to match the Pandas behavior.